packages/tour-kit/src/components/tour-kit-frame.tsx

```diff
@@ -22,8 +22,9 @@ const handleCallback = ( currentStepIndex: number, callback?: Callback ) => {
 };
 
 const TourKitFrame: React.FunctionComponent< Props > = ( { config } ) => {
+	const [ tourReady, setTourReady ] = useState( false );
 	const tourContainerRef = useRef( null );
-	const popperElementRef = useRef( null );
+	const [ popperElement, sePopperElement ] = useState< HTMLElement | null >( null );
```

Comment:
Recommended way for setting the popper reference (`useState` triggers a re-render): https://popper.js.org/react-popper/v2/